### ISSUE TITLE
Special Pixels

### ISSUE THREAD

**Issue by jlaura** [planetarypy/planetaryimage#55]
Should the [these](https://github.com/planetarypy/planetaryimage/blob/master/planetaryimage/specialpixels.py#L50) high special pixels be positive?

**Comment by wtolson** [comment]
I don't believe so. This is the relevant excerpt from SpecialPixel.h in the isis code base:

      // 2-byte signed special pixel values
      const short VALID_MIN2     = ((short)(-32752));
      const short NULL2          = ((short)(-32768));
      const short LOW_REPR_SAT2  = ((short)(-32767));
      const short LOW_INSTR_SAT2 = ((short)(-32766));
      const short HIGH_INSTR_SAT2 = ((short)(-32765));
      const short HIGH_REPR_SAT2  = ((short)(-32764));
      const short VALID_MAX2      = ((short)   32767);

**Comment by wtolson** [comment]
If you looking in the source, the file is `isis/src/base/objs/SpecialPixel/SpecialPixel.h`.
